### PR TITLE
chore: added the brand logos

### PR DESCRIPTION
**Pull request by rpandox**
This  PR adds the logos of various brands that the kec computer club colabrated with in the images section.

### CODE REVIEW

File: _layouts/home.html [hunk 1]
```diff
@@ -81,6 +81,9 @@
       </div>
     </div>
     <!-- Metrics Banner Ends Here -->
+    <!-- Brands Collabration section starts here -->
+    <section id="brands-collabration"></section>
+    <!-- Brands Collabration section ends here -->
```
Comment (rpandox) [inline _layouts/home.html:86]:
```suggestion

```

**Review by gaurovgiri — CHANGES_REQUESTED** [review]
change the name of the files with a consistent typography.
eg.
`Digital Gurkha.png` -> `digital_gurkha.png`

**Review by gaurovgiri — COMMENTED** [review]
change the name of the folder `Brands` -> `brands`

**Comment by rpandox** [comment]
Done!!!

**Pull request closed without merging**